classes/Product.php

```diff
@@ -6260,6 +6269,13 @@ public function getWsManufacturerName()
         return Manufacturer::getNameById((int) $this->id_manufacturer);
     }
 
+	public function getWsSpecificPrices()
```

Comment:
You mustn't use tabs :)
And could you please add phpdoc?

Reply:
My editor is configured with tabs and when i run php-cs-fixer on the file with the PrestaShop config it is changing lot of stuff in the file (like the short array syntax) :(

Reply:
Weird, are you sure you use the `.php_cs.dist` configuration?

```
[3.6.8] got@got:~/projects/prestashop/source/develop[develop]$ ./vendor/bin/php-cs-fixer fix classes/Product.php 
Loaded config default from "/home/got/projects/prestashop/source/develop/.php_cs.dist".
Using cache file "/home/got/projects/prestashop/source/develop/var/.php_cs.cache".
Paths from configuration file have been overridden by paths provided as command arguments.

Fixed all files in 0.001 seconds, 12.000 MB memory used
[3.6.8] got@got:~/projects/prestashop/source/develop[develop]$ git status
On branch develop
nothing to commit, working tree clean
```

Reply:
Yep:
$ php-cs-fixer fix classes/Product.php 
Loaded config default from "/Applications/MAMP/htdocs/tmp/PrestaShop/.php_cs.dist".
Using cache file "/Applications/MAMP/htdocs/tmp/PrestaShop/var/.php_cs.cache".
Paths from configuration file have been overridden by paths provided as command arguments.

Reply:
yes, this is what we are using too :thinking: 
https://github.com/PrestaShop/PrestaShop/blob/develop/.php_cs.dist

Reply:
Done with the file inside the var instead of root directory.